.gitignore

```diff
@@ -4,9 +4,13 @@
 
 /.idea/
 /.vscode/
-/gclient_config.py_entries
+gclient_config.py_entries
 /gh-pages/
 /target/
-/website/*.bundle.js
-/website/data.json
-/website/recent.json
+
+# We use something stronger than lockfiles, we have all NPM modules stored in a
+# git. We do not download from NPM during build.
+# https://github.com/denoland/deno_third_party
+yarn.lock
```

Comment:
I'd prefer to keep this actually

Reply:
Why? Also if you want to keep it, we can keep it in third_party - doesn't need to be in root.

Reply:
Actually I don't care enough to make the case for it.